CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 include(LLVMKompilePrelude)
 project (KLLVM CXX C)
 
+# Required to build the Python bindings as a static library that is then linked
+# into a shared library with the interpreter library generated by llvm-kompile.
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Do we need to do this globally or can we only do it for the targets that need it?